libs/remix-lib/src/execution/execution-context.ts

```diff
@@ -221,9 +245,11 @@ export class ExecutionContext {
     if (!infoCb) infoCb = () => {}
     if (context === 'vm') {
       this.executionContext = context
-      this.vms[this.currentFork].stateManager.revert(() => {
-        this.vms[this.currentFork].stateManager.checkpoint(() => {})
+      /*
```

Comment:
remove?

Reply:
I will add a comment here to explain why it's being commented.